src/transaction/locator_sr.h

```diff
@@ -139,6 +139,13 @@ extern int locator_multi_insert_force (THREAD_ENTRY * thread_p, HFID * hfid, OID
 				       HEAP_SCANCACHE * scan_cache, int *force_count, int pruning_type,
 				       PRUNING_CONTEXT * pcontext, FUNC_PRED_UNPACK_INFO * func_preds,
 				       UPDATE_INPLACE_STYLE force_in_place, bool dont_check_fk);
+// *INDENT-ON*
+extern int locator_put_classname_entry (THREAD_ENTRY * thread_p, const char *classname, const OID * class_oid);
+extern int locator_remove_classname_entry (THREAD_ENTRY * thread_p, const char *classname);
+extern int locator_update_classname_entry (THREAD_ENTRY * thread_p, const char *old_classname,
+					   const char *new_classname);
+
+// *INDENT-OFF*
```

Comment:
```suggestion
extern int locator_put_classname_entry (THREAD_ENTRY * thread_p, const char *classname, const OID * class_oid);
extern int locator_remove_classname_entry (THREAD_ENTRY * thread_p, const char *classname);
extern int locator_update_classname_entry (THREAD_ENTRY * thread_p, const char *old_classname,
					   const char *new_classname);
```

Reply:
this was because above and below functions are using c++ standard library, but I will move the newly defined function names below.